net-libs/usrsctp/usrsctp-20250331.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="A cross-platform userland SCTP stack"
+HOMEPAGE="https://github.com/sctplab/usrsctp"
+GIT_REV=881513ab3fc75b4c53ffce7b22b08e7b07fcc67a
+SRC_URI="https://github.com/sctplab/usrsctp/archive/${GIT_REV}.tar.gz -> ${P}.tar.gz"
```

Comment:
If you're taking a snapshot, you should version i as `0.9.5.0_pDATEOFLASTCOMMIT` so that it sorts _older_ than any future release.